esrally/driver/runner.py

```diff
@@ -1569,6 +1573,180 @@ def __repr__(self, *args, **kwargs):
         return "put-settings"
 
 
+class CreateTransform(Runner):
+    """
+    Execute the `create transform API https://www.elastic.co/guide/en/elasticsearch/reference/current/put-transform.html`_.
+    """
+
+    async def __call__(self, es, params):
+        transform_id = mandatory(params, "transform-id", self)
+        body = mandatory(params, "body", self)
+        defer_validation = params.get("defer-validation", False)
+        await es.transform.put_transform(transform_id=transform_id, body=body, defer_validation=defer_validation)
+
+    def __repr__(self, *args, **kwargs):
+        return "create-transform"
+
+
+class StartTransform(Runner):
+    """
+    Execute the `start transform API
+    https://www.elastic.co/guide/en/elasticsearch/reference/current/start-transform.html`_.
+    """
+
+    async def __call__(self, es, params):
+        transform_id = mandatory(params, "transform-id", self)
+        timeout = params.get("timeout")
+
+        await es.transform.start_transform(transform_id=transform_id, timeout=timeout)
+
+    def __repr__(self, *args, **kwargs):
+        return "start-transform"
+
+
+class StopTransform(Runner):
```

Comment:
Does it make sense to name the operation something like `wait-for-transform-completion`? otherwise in Rally report it looks like `stop-transform` took a while to complete..although it is clear from documentation.
```
|                                                 Min Throughput | stop-transform | 1.86915e+09 | docs/s |
|                                              Median Throughput | stop-transform | 4.36236e+09 | docs/s |
|                                                 Max Throughput | stop-transform | 6.24201e+09 | docs/s |
|                                        50th percentile latency | stop-transform |     1.95756 |     ms |
|                                        90th percentile latency | stop-transform |     3.07196 |     ms |
|                                        99th percentile latency | stop-transform |      5.7514 |     ms |
|                                      99.9th percentile latency | stop-transform |     15.3971 |     ms |
|                                       100th percentile latency | stop-transform |     96.2499 |     ms |
|                                   50th percentile service time | stop-transform |     1.95976 |     ms |
|                                   90th percentile service time | stop-transform |       3.073 |     ms |
|                                   99th percentile service time | stop-transform |      5.7514 |     ms |
|                                 99.9th percentile service time | stop-transform |     15.3971 |     ms |
|                                  100th percentile service time | stop-transform |     96.2499 |     ms |
|                                                     error rate | stop-transform |           0 |      % |
```